pom.xml

```
@@ -6,14 +6,73 @@
     <groupId>fish.payara.advisor</groupId>
     <artifactId>advisor-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Payara Advisor Tool</description>
+    <url>http://www.payara.fish</url>
+
+    <licenses>
+        <license>
+            <name>GPL Version 2 with Classpath Exception</name>
```

You've actually licensed this repo under CDDL + GPLv2 with Classpath exception.

```suggestion
            <name>CDDL + GPLv2 with classpath exception</name>
```

---

already applied